.github/workflows/linter.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Check Lint
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```

Comment:
What about the rest of the versions?

Reply:
> What about the rest of the versions?

From my knowledge, the Python version which `flake8` is installed on, has no affect on it's linting.
Running `flake8` with Python3.8 and Python3.6 should have no difference on the output given by `flake8`.

So I don't see the point of using different Python versions for the linting. 

I could be very wrong about this, so please do let me know if what I said is incorrect.

Reply:
all good